### ISSUE TITLE
xl 4.0.7 crashes

### ISSUE THREAD

**Issue by jcmarksafb** [smeighan/xLights#117]
xlights 4.0.7 crashes when I check the "output to lights" box when trying to run a sequence. It doesn't generate a failure report, it is a windows crash,

**Comment by stevegiron** [comment]
Update. Crashes for me too on 4.0.9

**Comment by AzGilrock** [comment]
I can't get it to crash.  Do you actually need hardware hooked up?

**Comment by smeighan** [comment]
i also am unable to crash it. dell i7 running windows 7. i have no hardware plugged in to my machine.

**Comment by stevegiron** [comment]
For your viewing pleasure, I have captured this phenomena. Not as exciting as a bigfoot sighting video, but it's proof this happens. 

http://www.stevesimages.com/xl4/issue117.mp4

As I watched the playback I saw 2 other issues, one of which Sean mentioned in today's tutorial.
One issue is the Model Preview breakout box did not show the effects until I clicked inside of it. That is new as previously it worked.

The second issue is between 11-12 seconds in the song not the video, there's left over artifacts on the tree from an effect I had previously placed and later deleted.

**Comment by AzGilrock** [comment]
Delete the fseq file and save again and the artifacts will go away.

**Comment by jcmarksafb** [comment]
Same problem here in 4.0.10 as soon as I click output to lights it crashes, same window message. Didn't have this problem in 4.0.8. but had it in 4.0.9 and .10
 


     On Tuesday, March 31, 2015 2:45 PM, stevegiron <notifications@github.com> wrote:
   

 For your viewing pleasure, I have captured this phenomena. Not as exciting as a bigfoot sighting video, but it's proof this happens. http://www.stevesimages.com/xl4/issue117.mp4As I watched the playback I saw 2 other issues, one of which Sean mentioned in today's tutorial.
One issue is the Model Preview breakout box did not show the effects until I clicked inside of it. That is new as previously it worked.The second issue is between 11-12 seconds in the song not the video, there's left over artifacts on the tree from an effect I had previously placed and later deleted.—

**Comment by jcmarksafb** [comment]
didn't get far enough to see if I had effects left over after making an effect shorter.
 


     On Tuesday, March 31, 2015 7:00 PM, John Marksbury <jcmarksafb@yahoo.com> wrote:
   

 Same problem here in 4.0.10 as soon as I click output to lights it crashes, same window message. Didn't have this problem in 4.0.8. but had it in 4.0.9 and .10
 


     On Tuesday, March 31, 2015 2:45 PM, stevegiron <notifications@github.com> wrote:
   

 For your viewing pleasure, I have captured this phenomena. Not as exciting as a bigfoot sighting video, but it's proof this happens. http://www.stevesimages.com/xl4/issue117.mp4As I watched the playback I saw 2 other issues, one of which Sean mentioned in today's tutorial.
One issue is the Model Preview breakout box did not show the effects until I clicked inside of it. That is new as previously it worked.The second issue is between 11-12 seconds in the song not the video, there's left over artifacts on the tree from an effect I had previously placed and later deleted.—
